src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -86,15 +86,11 @@ def __init__(self, cwd: Optional[str] = None):
         self.service = AnsibleRulebookService(cwd)
 
     def activate(
-        self,
-        activation: models.Activation,
-        deployment_type: str,
-        ws_base_url: str,
-        ssl_verify: str,
+        self, activation: models.Activation, deployment_type: str
```

Comment:
@mkanoor Well, I thought the same, but I say the same to you as to @hsong-rh. This method is called in several places and it would require more changes in the code and the tests. I want to keep small the PR and do this kind of refactors in small chunks. Do you agree to address it in later iterations?

Reply:
On the other hand, I think changing from Podman to Kubernetes would definitely be a new feature. I'm not sure about the scope of such a change.